wen-restart/src/wen_restart.rs

```diff
@@ -260,7 +262,7 @@ pub(crate) fn aggregate_restart_last_voted_fork_slots(
         // Because all operations on the aggregate are called from this single thread, we can
         // fetch all results separately without worrying about them being out of sync. We can
         // also use returned iterator without the vector changing underneath us.
-        let active_percent = last_voted_fork_slots_aggregate.active_percent();
+        let active_percent = last_voted_fork_slots_aggregate.min_active_percent();
```

Comment:
If there is no limit on percentage of churns stakes can change at each Epoch, it's possible. But it's only an issue if:
1. We have some slot OC'ed in the next Epoch (so 67% of the new Epoch stake confirmed it)
2. When 80% of the old Epoch stake returned LastVotedForks, we didn't have any slot in the new Epoch with 42% of the new Epoch stake
 
Even if this happens, the stake of the new Epoch has already been generated at the beginning of the old Epoch, so very soon the newly restarted cluster needs to cross the Epoch boundary so at that point they figured they don't have majority of the stake in the new Epoch to proceed.

Given the current stake distribution, I think it would be rare.